src/hyperion/external_interaction/callbacks/rotation/ispyb_callback.py

```diff
@@ -42,18 +51,26 @@ def activity_gated_start(self, doc: dict):
             )
             json_params = doc.get("hyperion_internal_parameters")
             self.params = RotationInternalParameters.from_json(json_params)
-            self.ispyb: StoreRotationInIspyb = StoreRotationInIspyb(
-                self.ispyb_config, self.params
+            dcgid = (
```

Comment:
nit: Personally I prefer `dcgid` to be called the longer `data_collection_group_id`

Reply:
In general I agree, but it depends on the scope - this name only lasts for one statement and is never accessed elsewhere. I changed it in `StoreRotationInIspyb` based on this comment though